tests/unit/test_http_session.py

```diff
@@ -102,7 +102,7 @@ def assert_request_sent(self, headers=None, body=None, url='/', chunked=False):
             url=url,
             body=body,
             headers=headers,
-            retries=False,
+            retries=ANY,
```

Comment:
Was there an issue with setting this to `Retry(False)`?

Reply:
Yes, `Retry` is a class without an `__eq__`.